app/src/main/java/com/example/calculator/activity/CalculatorActivity.kt

```diff
@@ -48,6 +49,24 @@ class CalculatorActivity : AppCompatActivity() {
         }
     }
 
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        outState.putSerializable("firstOperand", presenter.onSaveInstance().getFirstOperand())
```

Comment:
do not hardcode pls. you can put constants in a companion object

Reply:
Changed constants in code on onSaveInstance and onRestoreInstance on activity CalculatorActivity